modules/tensor_mechanics/src/materials/ComputePlaneFiniteStrain.C

```diff
@@ -11,21 +11,23 @@ InputParameters validParams<ComputePlaneFiniteStrain>()
 {
   InputParameters params = validParams<Compute2DFiniteStrain>();
   params.addClassDescription("Compute strain increment and rotation increment for finite strain under 2D planar assumptions.");
-  params.addCoupledVar("scalar_strain_zz", "Scalar variable containing the out-of-plane strain for generalized plane strain");
-  params.addCoupledVar("strain_zz", "Nonlinear variable containing the out-of-plane strain for plane stress");
+  params.addCoupledVar("scalar_strain", "Scalar variable for generalized plane strain");
+  params.addCoupledVar("variable_strain", "Nonlinear variable for plane stress");
```

Comment:
I'm struggling with the name "variable_strain", but I don't have a better alternative.
Regardless of what name we use, use this for the description: "Nonlinear variable for the out-of-plane strain (used to model plane stress conditions)"